tests/transformers/test_transformers_model_export.py

```diff
@@ -3470,7 +3566,55 @@ def test_save_model_card_with_non_utf_characters(tmp_path, model_name):
     assert txt == card_data.text
     data = yaml.safe_load(tmp_path.joinpath(_CARD_DATA_FILE_NAME).read_text())
     assert data == card_data.data.to_dict()
+def test_vision_pipeline_pyfunc_predict_with_kwargs(small_vision_model):
+    artifact_path = "image_classification_model"
+
+   
+    image_file_paths = ["https://raw.githubusercontent.com/mlflow/mlflow/master/tests/datasets/cat_image.jpg", "https://raw.githubusercontent.com/mlflow/mlflow/master/tests/datasets/tiger_cat.jpg"]  # Replace with actual image file paths
+    labels = ["tabby", "tabby cat", "tiger cat","Egyptian cat"]  # Replace with corresponding class labels
+
+    
+    inference_payload = json.dumps({
+        "inputs": {
+            "images": image_file_paths,  # Replace with the input field name for image classification
+        },
+        "params": {
+            "top_k": 2,  # Specify your image classification parameters here
+            "confidence_threshold": 0.8,  # Example parameter; adjust as needed
+        },
+    })
+
+    
+    expected_predictions = ["tabby", "tabby cat", "tiger cat","Egyptian cat"]  # Replace with expected class labels
```

Comment:
please remove these comments, plus similar comment here for validating the actual output labels; we don't need to do that.